web/wp-content/themes/borealis-2021/inc/custom-post-types/news-cpt.php

```diff
@@ -0,0 +1,49 @@
+<?php 
+
+if (!function_exists('pg_register_news_cpt')) {
+    /**
+     * Creates a Location Custom Post Type with a services custom taxonomy and custom metadata
+     */
+    function pg_register_news_cpt() {
+        // Arguments are:
+        // - slug.
+        // - single.
+        // - plural.
+        // - args (an optional array to overwrite any default settings).
+        $namespace = pg_get_namespace();
+        $News_CPT = new PG_Custom_Post_Type('news', 'News', 'News', array('icon' => 'dashicons-category', 'has_archive' => 'false', 'template' => [ [$namespace . '/news-meta-block']]));
+        $News_CPT->register();
+        $News_CPT->register_tags('research-area', 'Research-area', 'Research-areas');
```

Comment:
Going to add this to the custom taxonomy for Research Areas. We will need to remove it here as we are going to want to define that taxonomy on its own and have it shared across News, Research and Publications. Series and Content Type are fine as they're both specific to the Research Blogs